### PR TITLE
config file changes to support golden side

### PR DESCRIPTION
**Pull request by pragupta**

### CODE REVIEW

File: openpower/configs/hostboot/palmetto.config [hunk 1]
```diff
@@ -27,7 +27,9 @@ unset PCIE_HOTPLUG_CONTROLLER
 set CONSOLE
 set CONSOLE_AST2400
 
+#pnor flags
 set PNOR_IS_32MB
+unset PNOR_TWO_SIDE_SUPPORT
```
Comment (williamspatrick) [inline openpower/configs/hostboot/palmetto.config]: Default is 'n' already.  No need to specifically unset.

File: openpower/configs/hostboot/habanero.config [hunk 2]
```diff
@@ -38,3 +38,6 @@ set DISABLE_HOSTBOOT_RUNTIME
 unset SET_NOMINAL_PSTATE
 set HTMGT
 set START_OCC_DURING_BOOT
+
+#PNOR flags
+set PNOR_TWO_SIDE_SUPPORT
```
Comment (williamspatrick) [inline openpower/configs/hostboot/habanero.config]: Hostboot currently has as 'TWO_SIDE_SUPPORT'.  Are we ok with putting in as "PNOR_TWO_SIDE_SUPPORT" for now, knowing that it won't do anything until Hostboot is updated?

**Comment by williamspatrick** [comment]
I've merged the PNOR commits.  Please update pnor.mk's VERSION as well.

**Comment by pragupta** [comment]
This commit depends on 635e73dc21ed06c953826e21b176608c0d91707d
I have made the required changes to support PNOR_TWO_SIDE_SUPPORT in 635e73dc21ed06c953826e21b176608c0d91707d

**Comment by williamspatrick** [comment]
Resolved by #61

**Pull request closed without merging**